### PR TITLE
Bitbucket ci

### PR DESCRIPTION
**Pull request by BeyondEvil**
Adding Bitbucket Pipeline (CI) environment variables to the plugin.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Release Notes
 -------------
 
+(unreleased)
```
Comment (davehunt) [inline CHANGES.rst]: Version will most likley be 1.8.0

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,36,py,py3}, flake8
+envlist = py{27,36,37,py,py3}, flake8
```
Comment (davehunt) [inline tox.ini]: Could you also add this to `.travis.yml`?

File: pytest_metadata/plugin.py [hunk 3]
```diff
@@ -77,11 +78,11 @@ def pytest_configure(config):
...
 
-    for key, value in CONTINUOUS_INTEGRATION.items():
+    for provider in CONTINUOUS_INTEGRATION:
         [
-            config._metadata.update({v: os.environ.get(v)})
-            for v in value[1]
-            if os.environ.get(v)
+            config._metadata.update({var: os.environ.get(var)})
+            for var in provider
+            if os.environ.get(var)
```
[the remark this answers is not in the document]
Comment (davehunt) [reply pytest_metadata/plugin.py]: hmm.. I think the original idea may have been to output the key somewhere, but that would require a way to determine the environment, perhaps by using a unique variable from each. I'm fine with this change.

File: tox.ini (hunk 2 above)
Comment (BeyondEvil) [reply tox.ini]: Changed `deploy` to `3.7` as well. (Same as pytest-selenium).

File: .travis.yml [hunk 4]
```diff
@@ -10,14 +10,17 @@ jobs:
     - stage:
       python: 3.6
       env: TOXENV=py36
+    - stage:
+      python: 3.7
+      env: TOXENV=py37
```
Comment (davehunt) [inline .travis.yml]: In order to get this to work in Travis you need to set the distro. See pytest-selenium for an example:

```yml
    - stage:
      python: 3.7
      dist: xenial
      sudo: required
```

File: .travis.yml [hunk 5]
```diff
@@ -10,14 +10,17 @@ jobs:
...
     - stage:
       python: pypy
       env: TOXENV=pypy
     - stage:
       python: pypy3
       env: TOXENV=pypy3
     - stage: deploy
-      python: 3.6
+      python: 3.7
```
Comment (davehunt) [inline .travis.yml]: Same comment regarding 3.7. Also, can you use 3.7 for the flake8 job, too?

**Pull request merged**